src/helpers/flow-chart.ts

```diff
@@ -1,142 +1,131 @@
 // @ts-nocheck
```

Comment:
Are we god to remove `ts-nocheck` from this file now?

Reply:
I'll remove it in the other PR about removing todos\\